IanGRap/Squish
# Bugfix movement in gameover (#119)

Contributors: Austin

Goal:
- Fix the bug where players would keep moving if the game ended while they were moving
- Fix the bug in the player select lobby where if the player was moving when the "Begin Rave" screen started, the player would keep moving

What Changed:
- In PlayerInput, if the game state is pause, game over, match over or begin match (player select), reset the input  properties to false or 0, and don't listen for new input

How to test:
- In development, end the game while moving and watch the player continue to move
- Pull this branch and create the same scenario. The player should stop moving.
